### DISCUSSION TITLE
Accessing Directus System Collections via GraphQL

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#5498] [Archive]
I am unable to access (Directus System Collection) _Files_ via GraphQL (using Gatsby, but testing externally of it), but unable to query `files` as [documented](https://docs.directus.io/reference/api/system/files/#graphql). Note, using v9.0.0-rc.62.

- We are using authentication, and full permissions have been applied.
- Can access files via REST.

Is there any other configuration I might be missing?

**Comment by rijkvanzanten** [comment]
Are you using the system graphql endpoint? `/graphql/system`

**Reply by ghost** [reply]
No, I wasn't aware of it. I'll try it out. Is it in the docs? I haven't found a reference to it.

**Reply by ghost** [reply]
Confirming `/graphql/system` endpoint works!

**Reply by rijkvanzanten** [reply]
Clarified the docs a bit in d912f070f3c16a9ba70aa55b39d517634eb67e42 🙂
